ChatBot/app.py

```
@@ -0,0 +1,104 @@
+# This is a chatbot application
+import streamlit as st
+from agent.agent import HindiLearningAgent
+from models.embedding_model import FastTextEmbedding
```

Ruff (0.8.2)

`models.embedding_model.FastTextEmbedding` imported but unused

Remove unused import: `models.embedding_model.FastTextEmbedding`

(F401)